makefile

```diff
@@ -1,5 +1,6 @@
 DBGVIEW=*NONE
 BIN_LIB=MIGRATE
+CCSID=1144

```

Comment:
Perhaps this should be `*JOB` by default?

Reply:
Yes, absolutely. I should have updated it with a new commit.